src/main/scala/tech/cryptonomic/conseil/michelson/presenter/MichelsonPresenter.scala

```diff
@@ -0,0 +1,41 @@
+package tech.cryptonomic.conseil.michelson.presenter
+
+import tech.cryptonomic.conseil.michelson.dto._
+
+object MichelsonPresenter {
+
+  implicit class MichelsonSchemaPresenter(val self: MichelsonSchema) extends AnyVal {
+    def render(): String = {
+      val parameter = self.parameter.render()
+      val storage = self.storage.render()
+      val code = self.code.render()
+
+      s"""parameter $parameter;
+         |storage $storage;
+         |code { $code }""".stripMargin
+    }
+  }
+
+  implicit class MichelsonTypePresenter(val self: MichelsonType) extends AnyVal {
+    def render(): String = self match {
+      case MichelsonType(name, Seq()) => name
+      case MichelsonType(name, Seq(arg)) => s"($name ${arg.render()})"
+      case MichelsonType(name, Seq(arg1, arg2)) => s"($name ${arg1.render()} ${arg2.render()})"
```

Comment:
I have a question, which is probably because I'm not so familiar with `michelson` -> if this sequence contains 3 elements this will break. Should we handle it? Or just read first 2 of the elements like matching `arg1 :: arg2 :: _` ?

Reply:
Technically there is no way that there will be more than two parameters but since it can be generalized easily I did it.